client/my-sites/email/email-providers-stacked-comparison/provider-cards/google-workspace-card.tsx

```diff
@@ -158,8 +166,60 @@ const GoogleWorkspaceCard: FunctionComponent< EmailProvidersStackedCardProps > =
 		</div>
 	);
 
-	const onGoogleConfirmNewMailboxes = noop;
-	const onGoogleUsersChange = noop;
+	const [ googleUsers, setGoolgeUsers ] = useState( newUsers( selectedDomainName ) );
```

Comment:
Typo 😉:
```suggestion
	const [ googleUsers, setGoogleUsers ] = useState( newUsers( selectedDomainName ) );
```

Reply:
😄 Next pull request addressed it [here](https://github.com/Automattic/wp-calypso/pull/58761/files#diff-9d9202fc059e9bb4ee4a671444405c105288c06253b359ca52a4364647f60d93R145)